tutor/src/components/task-step/step-with-reading-content.cjsx

```diff
@@ -27,16 +27,20 @@ ReadingStepContent = React.createClass
   shouldOpenNewTab: -> true
   render: ->
     {id, courseDataProps, stepType} = @props
+
     {content_html, related_content} = TaskStepStore.get(id)
     {courseId} = @context.router.getCurrentParams()
 
     <div className="#{stepType}-step">
-      <ArbitraryHtmlAndMath
-        {...courseDataProps}
+      <div
         className="#{stepType}-content"
-        shouldExcludeFrame={@shouldExcludeFrame}
-        html={content_html} />
-      <RelatedContentLink courseId={courseId} content={related_content} />
+        {...courseDataProps}>
+        <RelatedContent contentId={id} {...related_content?[0]} />
+        <ArbitraryHtmlAndMath
+          className='book-content'
+          shouldExcludeFrame={@shouldExcludeFrame}
+          html={content_html} />
+      </div>
     </div>
```

Comment:
Throwing this out for debate:

I've been thinking that when a element has lengthy params, and flows onto multiple lines, and has children it's clearer to leave the trailing `>` indented the same as the start of the element like so:
```
<div
   className="foo"
   param1=1
   param2=2
>
  <a child />
</div>
```

That way you can tell where the open div tag starts and where the child is at.  For instance the above code is kinda hard to tell that `ReatedContent` is a child of the div since the `>` is nestled right after `{...courseDataProps}`.

Thoughts?